src/components/Calculator/ScoreTable.js

```diff
@@ -18,33 +30,65 @@ const ScoreTable = ({ subjects }) => (
             Kierunek
           </th>
           {/* <th scope='row'>
-            Przedmioty
-          </th> */}
+              Przedmioty
+            </th> */}
           <th scope='row'>
             Wynik
           </th>
+          <th>
+            2013
+          </th>
+          <th>
+            2014
+          </th>
+          <th>
+            2015
+          </th>
+          <th>
+            2016
+          </th>
         </tr>
-        {facultiesArray.map((faculty) =>
-          faculty.courses.map((course, courseIndex) =>
-            <tr key={courseIndex}>
-              {!courseIndex &&
-              <td rowSpan={faculty.courses.length}>
-                <strong>
-                  {faculty.name}
-                </strong>
-              </td>}
-              <td>
-                {course.name}
-              </td>
-              {/* <td>
-              {course.subjects.join(', ')}
-            </td> */}
-              <td>
-                <strong>
-                  {getCourseScore(course, faculty, subjects)}
-                </strong>
-              </td>
-            </tr>
+
+        {facultiesArray.map((faculty, facultyIndex) =>
+          faculty.courses.map((course, courseIndex) => {
+            const courseScore = getCourseScore(course, faculty, subjects)
+            const threshold = thresholdsArray[facultyIndex].courses[courseIndex]
+
+            return (
+              <tr key={courseIndex}>
+                {courseIndex
+                  ? <td style={{ display: 'none' }} />
+                  : <td rowSpan={faculty.courses.length}>
+                    <strong>
+                      {faculty.name}
+                    </strong>
+                  </td>}
+                <td>
+                  {course.name}
+                </td>
+                {/* <td>
+                {course.subjects.join(', ')}
+              </td> */}
+                <td>
+                  <strong>
+                    {courseScore}
+                  </strong>
+                </td>
+                <td>
+                  <Threshold courseScore={courseScore} threshold={threshold[2013]} />
```

Comment:
If you go for suggestion "What about getting available years from thresholds.json" then this can be also mapped.